.vsts-ci/phase.yml

```
@@ -1,5 +1,5 @@
 parameters:
-  queue: ubuntu-16.04
+  queue: 	ubuntu-18.04
```

extra space / tab here?
```suggestion
  queue: ubuntu-18.04
```